File: CleanCodeJava/app/src/main/java/io/chuumong/cleancodejava/data/model/CityData.java
package io.chuumong.cleancodejava.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by JongHunLee on 2017-02-22.
 */

public class CityData {

    @SerializedName("id")
    @Expose
    private String id;

    @SerializedName("name")
    @Expose
    private String name;

    @SerializedName("description")
    @Expose
    private String description;

    @SerializedName("background")
    @Expose
    private String background;

    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public String getBackground() {
        return background;
    }
}

File: CleanCodeKotlin/app/src/main/java/io/chuumong/cleancodekotlin/data/model/CityData.kt
package io.chuumong.cleancodekotlin.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by JongHunLee on 2017-02-23.
 */
data class CityData(
        @SerializedName("id")
        @Expose
        val id: String,

        @SerializedName("name")
        @Expose
        val name: String,

        @SerializedName("description")
        @Expose
        val description: String,

        @SerializedName("background")
        @Expose
        val background: String
)